rainbow-me/browser-extension
# Ledger legacy path

Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

- Added Legacy Ledger HD path on the AddByIndex sheet
- Refactored HW keychain to support different HD paths per accounts

## Screen recordings / screenshots

See how account 3 on MM matches index 2on rainbow
 
<img width="1623" alt="Screen Shot 2023-09-06 at 6 39 49 PM" src="https://github.com/rainbow-me/browser-extension/assets/1247834/1599534e-87a4-4fc1-bd46-4086be0ec7d8">



## What to test



- Make sure your ledger account works before & after switching branches (to test backwards compatibility)
- Add accounts by index for both hd paths after switching branches
- Send transactions from both paths
- Onboarding from scratch with both legacy and live paths
- Make sure trezor still works

## Final checklist
